E2E/pageObjects/game.page.ts

```diff
@@ -85,12 +89,25 @@ export = {
     I.switchToPreviousTab();
     I.seeElement(locator.chessBoard.board);
   }, 
+
   skipMove: () => {
     I.click(locator.buttons.skip);
     I.seeElement(locator.buttons.skipButtonHighlighted);
   },
+
   voteAndCheckThatVoteIsVisible:(chessPiece: ChessPiece, tile: string, value:string) => {
     game.vote(chessPiece, tile);
     game.voteIsVisible(chessPiece, tile, value);
   },
+
+  checkPlayerSP:(averageScore) => {
```

Comment:
Check what happens when player's average score is 1,33. It should return 1, yet your method returns 0